### PR TITLE
Add Chocolatey and GoFish to the quickstart docs

### LINKED ISSUE #161: distro packages?
From the original launch blog: https://azure.microsoft.com/en-us/blog/streamlining-kubernetes-development-with-draft/#comment-3338065825

### LINKED ISSUE #335: Make Draft Chocolatey
Similar to our Homebrew repository, Windows devs need some love.

https://chocolatey.org/

### PR DESCRIPTION
**Pull request by bacongobbler**
closes #161
closes #335

### CODE REVIEW

**Comment by bacongobbler** [comment]
Added note mentioning the Choco package is maintained by a community member and not by any of the core maintainers. Thanks @radu-matei for the suggestion. :)

File: docs/quickstart.md [hunk 1]
```diff
@@ -23,12 +23,40 @@ $ helm init
 ```
 
 ### Install and configure Draft
-- Download the Draft binary using Homebrew via the commands below or from the [official releases page](https://github.com/Azure/draft/releases)
+
+#### Standalone Binary
+
+Download the latest release of Draft from the [latest releases page](https://github.com/Azure/draft/releases/latest), unpack the binary and place it somewhere on your $PATH.
+
```
Comment (cnadolny) [inline docs/quickstart.md]: Perhaps add instructions on how to unpack it for linux systems? I wasn't able to get GoFish working on my machine, using Ubuntu for Windows.

1. Download the binary
2. Unpack it ( tar -zxvf draft-v0.14.1-linux-amd64.tar.gz )
3. Add it to path ( mv linux-amd64/draft /usr/local/bin/draft )

Comment (bacongobbler) [reply docs/quickstart.md]: That's a good suggestion. Thanks @cnadolny! I'll add those steps here.

Hmm, I have not tested GoFish under Windows Subsystem for Linux yet, but the native Windows client should work fine via https://www.gofi.sh/#install. If you have an issue I'd love to hear it at https://github.com/fishworks/gofish! Thanks for trying it out. :)

Comment (bacongobbler) [reply docs/quickstart.md]: added. Thanks!

**Pull request merged**